packages/runtime/container-runtime/src/dataStoreContext.ts

```diff
@@ -1000,6 +1022,20 @@ export class LocalFluidDataStoreContextBase extends FluidDataStoreContext {
         // Local data store does not have initial summary.
         return {};
     }
+
+    /** A context should only be marked as deleted when its a remote context */
+    public delete() {
+        sendGCTombstoneEvent(
+            this.mc,
+            {
+                eventName: "GC_Deleted_DataStore_Unexpected_Delete",
```

Comment:
This log should reflect somewhere that the delete is unexpected because its a local data store. Maybe in the name or add a property that says isLocal?